views/layouts/desktop.html

```diff
@@ -1,33 +1,18 @@
 <template name="desktop">
     <div class="wrapper">
+        <!-- Top Navbar -->
+        {{> topnav}}
         <!-- Sidebar -->
-        <nav id="sidebar" class="active">
-            {{> menu2}}
+        <nav id="sidebar">
+            {{> menu}}
         </nav>
         <!-- Content -->
         <div id="content">
-            {{> topnav}}
             {{> Template.dynamic template=main}}
             {{> Template.dynamic template=bottom}}
         </div>
     </div>
 </template>
-<template name="desktop2">
-  {{> sAlert}}
```

Comment:
Are you sure this isn't needed?

Reply:
This was copy of old view, I made copy before editing but forgot to delete in first PR.

Reply:
ok